crates/router/src/connector/stax/transformers.rs

```diff
@@ -374,7 +384,14 @@ impl TryFrom<types::RefundsResponseRouterData<api::Execute, RefundResponse>>
             .response
             .child_transactions
             .iter()
-            .filter(|txn| txn.total == item.data.request.refund_amount)
+            .filter(|txn| {
+                txn.total
+                    == utils::to_currency_base_unit_asf64(
```

Comment:
Do we get amount as minor units in response?

Reply:
The response from connector which is txn.total is returned in dollars hence I coverted item.data.request.refund_amount to dolllars.